postgresql/client.go

```diff
@@ -15,7 +15,7 @@ import (
 
 	"github.com/timescale/prometheus-postgresql-adapter/util"
 
-	_ "github.com/lib/pq"
+	"github.com/lib/pq"
```

Comment:
I think we only need to initialize the driver, so we should use blank identifier when importing.